src/main/java/io/openliberty/tools/intellij/LibertyExplorer.java

```diff
@@ -121,6 +124,8 @@ public static Tree buildTree(Project project, Color backgroundColor) {
             LibertyModuleNode node;
             try {
                 projectName = LibertyMavenUtil.getProjectNameFromPom(virtualFile);
+            } catch (IndexNotReadyException | ProcessCanceledException | CancellationException e) {
+                throw e;
```

Comment:
It should be used in any code that can be called by the language server thread. E.g. diagnostics, getting info for a quick fix. This looks very UI related so maybe we don't need this here. Unless that tree can be refreshed due to a lang. server starting up or similar.

Reply:
I couldn't find any way for these methods to be called from lsp4ij threads so I'll remove those catch statements. They were just added because the analysis would take too long.

Reply:
Down to 147 files.